packages/lib-classifier/src/store/PreviousAnnotations/PreviousAnnotations.js

```diff
@@ -0,0 +1,118 @@
+import asyncStates from '@zooniverse/async-states'
+import { GraphQLClient } from 'graphql-request'
+import { flow, types } from 'mobx-state-tree'
+
+export const caesarClient = new GraphQLClient('https://caesar.zooniverse.org/graphql')
+const CONSENSUS_SCORE_TO_RETIRE = 3
+const MINIMUM_VIEW_TO_RETIRE = 5
+
+const Annotation = types.model({
```

Comment:
Is this name confusing, given that we already have an `Annotation` model for classifications?

Reply:
Yes. Could we call caesar reduction related models for transcriptions `TranscriptionReductions` and `TranscriptionReduction` instead?

Reply:
Sure. Are these reductions only run for transcriptions?

Reply:
It'll only be for projects configured to use the transcription task and downstream with caesar to use the transcription specific reducers, so yes. We don't know how this full process would work with any other projects and task types and if the models will be the same.